Resources/Prototypes/Entities/Objects/Weapons/Guns/SMGs/smgs.yml

```
@@ -21,9 +21,9 @@
     - suitStorage
   - type: AmmoCounter
   - type: Gun
-    minAngle: 2
-    maxAngle: 16
-    fireRate: 8
+    minAngle: 11 #220 SG Angle Updates
+    maxAngle: 20 #220 SG Angle Updates
+    fireRate: 7.5 #220 SG Angle Updates
```

```suggestion
    #220 SG Angle Updates begin
    minAngle: 11 
    maxAngle: 20
    fireRate: 7.5
    #220 SG Angle Updates end
```

И по такому образцу все остальные **многострочные** изменения